Application/EdFi.Ods.SwaggerUI/EdFi.Ods.SwaggerUI.csproj

```diff
@@ -83,8 +83,9 @@
       <HintPath>..\packages\Owin.1.0\lib\net40\Owin.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="swagger-ui, Version=3.0.7412.27465, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\EdFi.swagger-ui.3.4.0\lib\net48\swagger-ui.dll</HintPath>
+    <Reference Include="swagger-ui, Version=3.0.7407.39704, Culture=neutral, PublicKeyToken=null">
```

Comment:
Our base swagger-ui verison went down?  From 3.0.7412 to 3.0.7407?  Is this an older net core build?